src/main/java/liuni/resources/LiUniResource.java

```diff
@@ -0,0 +1,53 @@
+package liuni.resources;
+
+import liuni.TwitterStatus;
+import liuni.TwitterTimeline;
+import liuni.api.ErrorModel;
+import liuni.api.TwitterTimelineModel;
+import com.codahale.metrics.annotation.Timed;
+import liuni.api.TwitterTweetModel;
+import org.json.simple.JSONObject;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+
+@Path("/api/1.0/twitter")
+@Produces(MediaType.APPLICATION_JSON)
+public class LiUniResource {
+
+    public LiUniResource() {}
+
+    @Path("/timeline")
+    @GET
+    @Timed
+    public JSONObject fetchTimeline() {
+        try {
+            TwitterTimeline twitterTimeline = new TwitterTimeline();
+            JSONObject[] timeline = twitterTimeline.getTimelineJson();
```

Comment:
what are the pros/cons of having all of the methods return JSONObject/+[]

Reply:
Pro: Easier for the frontend to digest and work with the information more readily, no converting between typing like String to JSON
Con: JSON files may include more information than the frontend cares about (I did not think that would be an issue in this case because you had repeatedly said you wanted all the status information, but if the app was only going to display the user who posted the status and the tweet itself then all the other info is just clutter).

Reply:
Apparently there is the additional con of not being able to change the status code easily.